exercicios/modulo-01/ex016beatriz/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML CSS 1</title>
+</head>
+<body>
+    <H1>Prática dos exercícios do Módulo 1 de HTML e CSS do Curso em Vídeo </H1>
+    <p>Quando eu quiser <br>quebrar uma linha sem colocar outra tag no código, preciso adicionar a tag br.</p> <!-- uma tag br para cada quebra -->
+    <p>Para adcionar símbolos como &trade; é necessário colocar  o logograma "&" antes da escrita e o sinal ";" depois, ambos sem espaço. Já para emoji, é necessário colocar "&#x" antes do código disponível no site <a href="https://emojipedia.org/" target="_blank"> emojipedia</a> . &#x1F609</p>
+    <p>Não devemos esquecer dos direitos autorais de imagens, vídeos e áudios. É necessário buscar nos lugares corretos como: <img src="img/leoa_e_filhote.jpg" alt="Foto da mãe leoa deitada com seu filhote"> <em>essa imagem foi retirada do google imagens e eu escolhi a opção de com direito criativo.</em> <img src="img/leoa_rugindo_l1500r50.jpg" alt="Leoa rugindo"> <em>já essa foto é do site <a href="https://www.pexels.com/pt-br/" target="_blank" >pexels</a></em>, também foi alterado o tamanho e resolução pelo <a href="https://www.gimp.org/" target="_blank">Gimp.</a></p>
+    
```

Comment:
voce pode colocar as imagens em uma pasta, para ficar mais organizado e intuitivo quando for fazer alguma atualização.